src/test/resources/Condition/create-chronic-lung-disease-infectious-disease-disorder.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "77a8826d-4397-4937-8cca-de90008d3368",
+    "meta": {
+      "profile": [
+        "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/complications-covid-19"
```

Comment:
@af-dpe  are you sure this is the correct example?

Reply:
as agreed please remove TC and related example JSON

Reply:
TC and json is removed